tests/lean/printStructure.lean

```diff
@@ -4,70 +4,87 @@
 
 /- Structure -/
 /--
-info: structure Prod.{u, v} : Type u → Type v → Type (max u v)
+info: structure Prod.{u, v} (α : Type u) (β : Type v) : Type (max u v)
 number of parameters: 2
-constructor:
-Prod.mk : {α : Type u} → {β : Type v} → α → β → α × β
 fields:
-fst : α
-snd : β
+  Prod.fst : α
+  Prod.snd : β
+constructor:
+  Prod.mk.{u, v} {α : Type u} {β : Type v} (fst : α) (snd : β) : α × β
 -/
 #guard_msgs in
 #print Prod
 
 /- Class -/
 /--
-info: class Inhabited.{u} : Sort u → Sort (max 1 u)
+info: class Inhabited.{u} (α : Sort u) : Sort (max 1 u)
 number of parameters: 1
-constructor:
-Inhabited.mk : {α : Sort u} → α → Inhabited α
 fields:
-default : α
+  Inhabited.default : α
+constructor:
+  Inhabited.mk.{u} {α : Sort u} (default : α) : Inhabited α
 -/
 #guard_msgs in
 #print Inhabited
 
 /- Structure with private field -/
 /--
-info: structure Thunk.{u} : Type u → Type u
+info: structure Thunk.{u} (α : Type u) : Type u
 number of parameters: 1
-constructor:
-Thunk.mk : {α : Type u} → (Unit → α) → Thunk α
 fields:
-private fn : Unit → α
+  private _root_.Thunk.fn : Unit → α
```

Comment:
Is there a test anywhere of `private` _constructor_ printing?

Reply:
There must not be, because there's nothing to print `private` in front of constructors. I'll add it.